src/main/scala/nl.knaw.dans.easy.stage/Settings.scala

```diff
@@ -37,7 +37,9 @@ case class Settings(ownerId: String,
                     databaseUrl: String,
                     databaseUser: String,
                     databasePassword: String,
-                    licenses: Map[String, File]) {
+                    licenses: Map[String, File],
+                    stageDatasetVersion: String,
+                   ) {
```

Comment:
This requires also a change to `eas-ingest-flow` just to add the version to the `remarkerId`. Since that field was intended to contain a `userId` (not an application) the issue specs might need a change.